src/js/utils.js

```diff
@@ -0,0 +1,59 @@
+export function capitalizeFirstLetter(string) {
+	return string[0].toUpperCase().concat(string.slice(1));
+}
+
+export function getID(url) {
+	const numbersRegex = /\d/g;
+	return url.match(numbersRegex).slice(1).join('');
+}
+
+// export function eggGroupsToString(obj) {
+//     return obj.map(innerObj => parseFromHyphen(innerObj.name)).join(', ');
+// }
+
+// export function abilitiesToString(obj) {
+//     return obj.map(innerObj => parseFromHyphen(innerObj.ability.name)).join(', ');
+// }
+
+function parseFromHyphen(string) {
+	return capitalizeFirstLetter(string.replaceAll('-', ' '));
+}
+
+export function objectPropsToString(obj, prop) {
+	return obj.map((innerObj) => {
+		const value = prop === 'name' ? innerObj.name : innerObj.ability.name;
+		return parseFromHyphen(value);
+	}).join(', ');
+}
+
+export function convertMetersToFeetAndInches(meters) {
+	const totalInches = meters * 39.3701;
+	const feet = Math.floor(totalInches / 12);
+	const inches = Math.round(totalInches % 12);
+	return `${feet}'${inches}"`;
+}
+
+export function convertKgToLb(kg) {
+	const lb = kg * 2.20462;
+	return lb;
+}
+
+export function parseToThreeDigits(number) {
+	if (number < 10) {
+		return `00${number.toString()}`;
+	} if (number < 100) {
+		return `0${number.toString()}`;
+	} return number.toString();
+}
+
+export function parseFromSnake(string) {
```

Comment:
`parseFromSnakeConvention(pokemonStat)`?

Reply:
I think that this function has a much more general use, is it worth adding such specific parameters?